src/Stack.php

```diff
@@ -130,8 +130,9 @@ public function toArray(): array
     }
 
     /**
-     *
+     * @inheritDoc
```

Comment:
I don't think this was necessary. Not only is it unrelated to this PR, but documentation is inherited automatically without this annotation. And not that it should matter since it should be removed, but this tag is also an inline tag, thus must be enclosed in curly braces when it _is_ needed.

Reply:
All docblocks have been restored to their original state.